proto/get_deps.sh

```diff
@@ -2,7 +2,7 @@
 
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
-TM_VERSION=v0.34.14
+TM_VERSION=v0.34.21
```

Comment:
sneaky

Reply:
My bad. Reverted this change and created: https://github.com/celestiaorg/optimint/pull/547